### ISSUE TITLE
Attempt to configure "Terminal Chat" on second machine results in invalid Azure OpenAI endpoint error.

### ISSUE THREAD

**Issue by akulbe** [microsoft/terminal#16442]
### Windows Terminal version

1.20.3401.0

### Windows build number

22635.2841

### Steps to reproduce

1. I have "Terminal Chat" successfully configured on one machine already.
2. Open up Terminal, go to Settings > Terminal Chat, add the endpoint URL and Key 2.
3. Save Settings and close Terminal.
4. Reopen Terminal and go to Terminal Chat.
5. Attempt to enter prompt.
6. Be greeted with a message saying the endpoint I provided is not a valid Azure OpenAI endpoint.

![image](https://github.com/microsoft/terminal/assets/123279/4e099d00-0ad3-45cc-856e-35b44bf4528f)
![image](https://github.com/microsoft/terminal/assets/123279/37a8e95d-0a1f-4762-a7f4-40ddc78e0391)

### Expected Behavior

![image](https://github.com/microsoft/terminal/assets/123279/ec10fdd6-e8c3-4f97-b58a-97140691bcdd)

I was expecting similar results as this screenshot shows, where the Terminal is _actually_ connecting to, and interacting with, the LLM.

### Actual Behavior

Received the "Not a valid endpoint" error message

**Comment by g0t4** [comment]
I can't configure Terminal Chat to work either, I get a different warning `Resource not found` and I have verified the endpoint works when I make a curl call... one concern I have is, what is the deployment name supposed to be? The docs don't list a name to use and forming an API request requires a deployment name (unless it does some model lookup and uses first matching model?)... the docs show `TerminalSuggestions` model deployment name but that doesn't work either. I am using canary `1.22.1661.0`

**Comment by g0t4** [comment]
FYI my Azure OpenAI service endpoint now uses `.openai.azure.com` and not `.cognitiveservices.azure.com` so maybe check your Azure portal for a new endpoint?

**Comment by PankajBhojwani** [comment]
Make sure you are using the correct endpoint! The one in your screenshot is not the endpoint for your deployment, you need to go into your deployment's "playground" in azure ai studio and then click "view code" to find the correct one:


![Image](https://github.com/user-attachments/assets/81bf038e-dc60-4ab5-96e8-64aacda2699d)

**Comment by PankajBhojwani** [comment]
Agreed that it is quite difficult to find the correct endpoint unfortunately, even in my screenshot the correct endpoint is **not** the one in the textbox, it is the one in the `curl` command (the one that is blanked out but pointed to with the red arrow)

**Comment by DHowett** [comment]
We recently announced a new "Intelligent Terminal", which is the evolution of Terminal Chat. It's currently positioned as a totally separate product to make it clear that it's not ready to roll out to all of our many users. :)

Check it out at http://aka.ms/intelligent-terminal
